src/main/resources/sql/create.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE IF NOT EXISTS "shop" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255)        NOT NULL,
+  "address" VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "customer" (
+  "id"      SERIAL PRIMARY KEY,
+  "name"    VARCHAR(255) NOT NULL,
+  "surname" VARCHAR(255) NOT NULL,
+  "age"     INT2         NOT NULL,
+  "shop_id" INT          NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "purchase" (
+  "id"          SERIAL PRIMARY KEY,
+  "date"        TIMESTAMP NOT NULL,
+  "customer_id" INT       NOT NULL,
+  UNIQUE ("date","customer_id")
+);
+
+CREATE TABLE "product" (
+  "id"          SERIAL PRIMARY KEY,
+  "name"        VARCHAR(255) NOT NULL,
+  "price"       MONEY        NOT NULL
+);
+
+CREATE TABLE "purchase_detail" (
+  "id"          SERIAL PRIMARY KEY,
+  "quantity"    INT2  NOT NULL,
+  "price"       MONEY NOT NULL,
+  "shop_id"     INT   NOT NULL,
+  "purchase_id" INT   NOT NULL,
+  "product_id"  INT   NOT NULL
+);
+
+ALTER TABLE "customer"
```

Comment:
Why you add foreign keys in the end of script instead of in CREATE statements?

Reply:
I made it this way just for the practice... Will be fixed with next commit. Thanks!

Reply:
I think it's okay to add foreign keys at the bottom otherwise it will be necessary to reorder "create table" statements in order to compile sql file. But it's just my thoughts, I've never read about this

Reply:
Shop id needed in this table? Do we have goods of different shops in one check in real life?